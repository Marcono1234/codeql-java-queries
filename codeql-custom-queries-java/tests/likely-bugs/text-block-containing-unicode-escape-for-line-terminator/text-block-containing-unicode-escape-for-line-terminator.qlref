./likely-bugs/text-block-containing-unicode-escape-for-line-terminator.ql
