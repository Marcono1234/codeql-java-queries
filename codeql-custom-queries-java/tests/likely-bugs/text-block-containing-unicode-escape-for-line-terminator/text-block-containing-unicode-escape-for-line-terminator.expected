| Test.java:3:9:5:11 | """\n        \\u000a \\u000A \\u000d \\u000D\n        """ | Contains Unicode escape `\\u000A`; should use `\\n` |
| Test.java:3:9:5:11 | """\n        \\u000a \\u000A \\u000d \\u000D\n        """ | Contains Unicode escape `\\u000D`; should use `\\r` |
| Test.java:3:9:5:11 | """\n        \\u000a \\u000A \\u000d \\u000D\n        """ | Contains Unicode escape `\\u000a`; should use `\\n` |
| Test.java:3:9:5:11 | """\n        \\u000a \\u000A \\u000d \\u000D\n        """ | Contains Unicode escape `\\u000d`; should use `\\r` |
| Test.java:7:9:8:11 | """\\u000A some text\n        """ | Contains Unicode escape `\\u000A`; should use `\\n` |
